src/components/chart.svelte

```diff
@@ -39,22 +39,24 @@
         let minYValue = Math.min(...allValues)
         let maxYValue = Math.max(...allValues)
 
+        drawYLabels(minYValue, maxYValue, 10)
```

Comment:
Shall we call this within redrawGraph function instead?

Reply:
Chart component should accept `ticks` and `let Y_AXIS_TICKS = ticks || 10` should be globalised.

Reply:
Sure. It needs the min and max Y values to be supplied or could create a function that returns them (as they are required in drawGraph as well.
`
function getMinandMaxValues(values) {
        const allValues = valueSet.flat()
        let min = Math.min(...allValues)
        let max = Math.max(...allValues)
        return {min, max}
}
`